docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -211,7 +210,7 @@ supports Container Network Interface (CNI) based networks (and does not support
 
 Several projects provide Kubernetes pod networks using CNI, some of which also
 support [Network Policy](/docs/concepts/services-networking/networkpolicies/). See the [add-ons
-page](/docs/concepts/cluster-administration/addons/) for a complete list of available network add-ons.
+page](/docs/concepts/cluster-administration/addons/) for a complete list of available network add-ons. IPv6 support was added in [CNI v0.6.0](https://github.com/containernetworking/cni/releases/tag/v0.6.0), so for IPv6 based network please use CNI version 0.6.0 or higher. [CNI bridge](https://github.com/containernetworking/plugins/blob/master/plugins/main/bridge/README.md) and [local-ipam](https://github.com/containernetworking/plugins/blob/master/plugins/ipam/host-local/README.md) are the only supported drivers in 1.9.
```

Comment:
substitute "supported drivers in 1.9" with "supported IPv6 network plugins in 1.9"

delete ", so for IPv6 based network please use CNI version 0.6.0 or higher"